src/app/monthly-calc/monthly-calc.component.html

```diff
@@ -67,15 +67,20 @@
       </div>
       <mat-form-field class="input-wrap">
         <mat-label>Monthly Payment</mat-label>
-        <input matInput formControlName="monthlyPayment" />
+        <input
+          matInput
+          formControlName="monthlyPayment"
+          (keyup.enter)="monthlyPayment()"
```

Comment:
is this only trigger for calculating monthly payment ?
i see button is being removed and 'apply' button does not have any callback for calculations
if so, this is considered bad UX, add calculation trigger on 'apply' button

Reply:
Yes, there was button "calculate" and "apply" - which will be necessary for further action when user decides to apply. So if i apply the calculation to "apply" button, need to have other way how to actually apply then. I took down "calculate" because someone pointed out its has no function since the result is displayed in responsive way in input. Taking all this in consideration - should i still apply this method to "apply" button or return the "calculate" button in its previous glory?

Reply:
Ignas proposal - can we make it so it doesn't calculate on value change but when we press the calculate button? And then after pressing that, an apply button appears?

Reply:
sounds good